docs/sphinx/source/fsd/FSD-Appendices.md

```diff
@@ -20,6 +20,7 @@
 | survey\_date\_end         | String      | See S-101 Survey date end                                                                                                        |
 | source\_institution       | String      | e.g. "NOAA Office of Coast Survey"                                                                                               |
 | source\_survey\_id        | String      | e.g. "H99999"                                                                                                                    |
+| source\_survey\_index     | UInt64      | A value of 0 indicates the index is uninitialized or unused.                                                                     |
```

Comment:
Does 'source_survey_index' have the same intended usage of S-102 'id' in featureAttributeTable? If yes, why using a different name and type (UInt64 vs. integer unsigned 32-bit)?

CC: @anthonyklemm

Reply:
I suspect the answer is "yes", although I have not talked with Anthony about the S-102 field recently.  The cost of the field is low because the tables will (likely) be relatively small so I made it field larger than the current 32 bit space we use in NBS to accommodate more survey IDs in the future.

CC: @barry-gallagher

Reply:
@GlenRice-NOAA Is it okay to proceed with merging this and implementing it in API code, tests, and example code?

Reply:
integer unsigned 32-bit is okay with me.

Reply:
> integer unsigned 32-bit is okay with me.

@GlenRice-NOAA Can you make this change and push? Then I can merge and implement this within the v2 branch (since this is from your fork, which I don't have access to). Thanks.